tests/scaletest/scale_test.go

```diff
@@ -161,7 +277,26 @@ func PoolVerification(t *testing.T) bool {
 	return false
 }
 
+/* Verifies if all requires DNS A records are created in the DNS VS or not */
+func DNSARecordsVerification(t *testing.T, hostNames []string) bool {
```

Comment:
The logic of this function may has to be tweaked a bit. E.g. if there were 2 shard VSes initially, then another 2 shard VSes gets created during test run, then len(diffString) would be 4, and len(diffString)-val) would be 0. The following condition would not be true.
    if (len(diffString)-val) == initialNumOfFQDN || len(diffString) == initialNumOfFQDN

Reply:
Nice catch! Making changes to the logic